SimplePayment.Test/UnitTest1.cs

```diff
@@ -7,12 +12,67 @@ public class Tests
         [SetUp]
         public void Setup()
         {
+            TestBillingDetailModel();
+            TestPaymentResponseModel();
+        }
+        public string RemoveWhiteSpace(string json)
+        {
+            return Regex.Replace(json, @"(""[^""\\]*(?:\\.[^""\\]*)*"")|\s+", "$1");
         }
 
         [Test]
-        public void Test1()
+        public void TestBillingDetailModel()
         {
-            Assert.Pass();
+            var billingDetailsJson = @"{
+                ""Name"" : ""Teszt B�la"", 
+                ""Company"" : ""Teszt Kft."", 
+                ""Country"" : ""Magyarorsz�g"", 
+                ""City"" : ""Szeged"", 
+                ""State"" : ""Csongr�d"", 
+                ""Zip"" : ""6722"", 
+                ""Address"" : ""Teszt utca 7."", 
+                ""Address2"" : ""2 / 3"", 
+                ""Phone"":""36701234567""
+            }";
+            var billingDetails = JsonConvert.DeserializeObject<BillingDetails>(billingDetailsJson);
+            var billingDetailsSerialized = JsonConvert.SerializeObject(billingDetails);
+
+            Assert.IsTrue(
+                string.Equals(billingDetails.Name, "Teszt B�la") ||
+                string.Equals(billingDetails.Company, "Teszt Kft.") ||
+                string.Equals(billingDetails.Country, "Magyarorsz�g") ||
+                string.Equals(billingDetails.City, "Szeged") ||
+                string.Equals(billingDetails.State, "Csongr�d") ||
+                string.Equals(billingDetails.Zip, "6722") ||
+                string.Equals(billingDetails.Address, "Teszt utca 7.") ||
+                string.Equals(billingDetails.Address2, "2 / 3") ||
+                string.Equals(billingDetails.Phone, "36701234567")
+            );
```

Comment:
Na itt viszont, && kéne || helyett, vagy amúgy lehet mindegyik külön Assert. Én külön raknám őket, de így is rendben van. Csak && legyen azért, mert így most ha csak 1 érték jó, akkor is átmegy a teszten, közben meg a többi lehet rossz.